tests/db/test_csv_db.py

```diff
@@ -0,0 +1,132 @@
+import os
+import unittest
+
+from remoteappmanager.db.csv_db import (
+    CSVApplication, CSVApplicationPolicy, CSVUser, CSVAccounting)
+
+from tests.db.abc_test_interfaces import ABCTestDatabaseInterface
+from tests.temp_mixin import TempMixin
+
+
+class GoodTable:
```

Comment:
I wonder if it would be better just to have three files in the fixtures directory instead.